scripts/src/main/resources/scripts/setup-docker

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+docker_check=$(docker --version)
+if [[ $docker_check != *"Docker version"* ]];then
+    echo "Installing Docker..."
+    sudo apt-get update -y
+    sudo apt-get install -y apt-transport-https ca-certificates curl software-properties-common
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+    sudo apt-key fingerprint 0EBFCD88
+    sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+    sudo apt-get update -y
+    sudo apt-get install -y docker-ce
+    sudo usermod -aG docker $USER
+    echo Docker installed successfully.
+    docker --version
```

Comment:
please do docker --verion after fi so it will in any case be outputted

Reply:
Done